coderd/rbac/partial.go

```diff
@@ -0,0 +1,144 @@
+package rbac
+
+import (
+	"context"
+
+	"golang.org/x/xerrors"
+
+	"github.com/open-policy-agent/opa/rego"
+)
+
+type PartialAuthorizer struct {
+	// PartialQueries is mainly used for unit testing to assert our rego policy
+	// can always be compressed into a set of queries.
+	PartialQueries  *rego.PartialQueries
```

Comment:
my preference would be to make these testing only fields not exported, and move the unit tests that need them into the `rbac` package.

Reply:
Yea, this just means I need to move `authz_test` to `authz_internal_test`.II just did that, and unexported all the fields.